frontend/src/metabase/components/Card/Card.tsx

```diff
@@ -15,11 +15,13 @@ type CardProps = {
 
 const Card = styled.div<CardProps>`
   background-color: ${(props) =>
-    props.dark ? color("text-dark") : color("bg-white")};
+    props.dark
+      ? "var(--mb-color-background-inverse)"
+      : "var(--mb-color-background)"};
   border: 1px solid
-    ${(props) => (props.dark ? "transparent" : "var(--mb-color-bg-medium)")};
+    ${(props) => (props.dark ? "transparent" : "var(--mb-color-border)")};
   ${(props) => props.dark && `color: white`};
-  border-radius: 6px;
+  border-radius: 8px;
```

Comment:
we have some nice border radius presets now
```suggestion
  border-radius: var(--mantine-radius-md);
```